src/modules/UI/scenes/WalletList/WalletList.ui.js

```diff
@@ -241,7 +241,7 @@ export default class WalletList extends Component<Props, State> {
                   styles.walletsBoxHeaderAddWallet,
                   {width: 41}
                 ]}
-                  onPress={Actions.createWallet}>
+                  onPress={Actions.createWalletName}>
```

Comment:
Going forward. Avoid directly using the scene name in an Action, use the constant instead as this can be checked with eslint.

`Actions[Constants.CREATE_WALLET_NAME]`